src/configurations/destinations/ga4_v2/db-config.json

```diff
@@ -19,6 +19,7 @@
       "eventFilteringOption",
       "extendPageViewParams",
       "piiPropertiesToIgnore",
+      "overrideClientAndSessionId",
```

Comment:
shouldn't it be added in defaultConfig too ?

Reply:
We are not using it in cloud mode